src/components/GameImg.jsx

```diff
@@ -0,0 +1,39 @@
+import { React, useState, useEffect, useRef } from "react";
+import styled from "styled-components";
+
+const ImgContainer = styled.div`
+  display:flex;
+  width:100%;
+  flex-direction:column;
+  text-align:center;
+`;
+const StyledImg = styled.img`
+  width: 100%;
+  height:80%;
+  object-fit: cover;
+  &:hover {
+    transform: scale(1.1); // 이미지 1.1배 확대
+    cursor: pointer;
+    transition: 0.5s; //부드럽게 확대
+  }
+`;
+const Desc = styled.h1`
+  position: relative;
+  top: -50%;
+  font-size:40px;
+  word-breka:keep-all;
+  color: white;
+  text-shadow: 3px 3px 3px black; //글자 잘 안 보여서 shadow 설정
+`;
+//게임에 사용되는 이미지 컴포넌트
+function GameImg({ src, name, onClick }) {
+  //src: 이미지 , name: 사진에 대한 이름, onCLick: 온클릭 함수
+
+  return (
+    <ImgContainer onClick={onClick}>
+      <StyledImg src={src} name={name} />
```

Comment:
```suggestion
      <StyledImg src={src} alt={name} />
```
`name={name}`으로 준 이유가 있을까? alt로 대신해도 좋을 것 같아 !!

Reply:
> `name={name}`으로 준 이유가 있을까? alt로 대신해도 좋을 것 같아 !!

아 마저! 이게 다른 컴포넌트들에서 name으로 받아오다보니까 무심코 이미지도 name으로 받아왔던 것 같아!! 이것도 바꿔놔야겠다 히히